engines/director/window.h

```diff
@@ -129,7 +129,7 @@ class Window : public Graphics::MacWindow, public Object<Window> {
 	DirectorEngine *getVM() const { return _vm; }
 	Archive *getMainArchive() const { return _mainArchive; }
 	Movie *getCurrentMovie() const { return _currentMovie; }
-	Common::String getCurrentPath() const { return _currentPath; }
+	Common::String getCurrentPath() const { return Common::String::format("%c%s", _vm->_dirSeparator, _currentPath.c_str()); }
```

Comment:
Are you sure it will not be turned into a path from the FS root?

Reply:
Replaced this with an approach that just changes "the pathName"